plugins/bigbluebutton/bbb_usage

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+"""
+
+=head1 NAME
+
+bbb - monitor Bigbluebutton server (users, rooms, videostreams ...)
+
+=head1 APPLICABLE SYSTEMS
+
+Bigbluebutton server
+
+=head1 CONFIGURATION
+
+No configuration is needed. This script will fetch the secret from the local
+bbb instance
+
+https://docs.bigbluebutton.org/dev/api.html
+
+=head1 AUTHOR
+
+Copyright 2021 Henning Rieger <age@systemausfall.org>
+
+With many thanks to the original Author Bernd Wurst (bwurst.org)
+
+=head1 LICENSE
+GNU General Public License v3.0 or later
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+=cut
+"""
+
+import sys
+import subprocess
+import re
+import hashlib
+import urllib.request
+from urllib.error import URLError, HTTPError
+import os
+from xml.etree import ElementTree
+
+bbbconf_path = "/usr/bin/bbb-conf"
```

Comment:
If you want to provide a fancy interface, you could take this path from the environment (e.g. `bbbconf_path = os.getenv("BBBCONF_BIN", "/usr/bin/bbb-conf")`) and describe this in the *Configuration* section.